.github/workflows/builder-scan.yml

```diff
@@ -1,11 +1,12 @@
 name: Builder Scan
 
 on:
+  workflow_dispatch:
```

Comment:
it needs a value?? after the `:`

Reply:
Pretty sure it doesn't

Reply:
can  remove `:` https://docs.github.com/en/actions/reference/events-that-trigger-workflows#example as per this example

Reply:
Have a look at a working example in .github/workflows/docker-base.yml